ingest/src/main/scala/hydra.ingest/services/TopicBootstrapActor.scala

```diff
@@ -51,9 +52,23 @@ class TopicBootstrapActor(config: Config,
 
   def active: Receive = {
     case InitiateTopicBootstrap(topicMetadataRequest) =>
+      val requestor = sender
+
       TopicNameValidator.validate(topicMetadataRequest.streamName) match {
         case Success(_) =>
-          initiateBootstrap(topicMetadataRequest) pipeTo sender
+          val ingestFuture = ingestMetadata(topicMetadataRequest)
+
+          val registerSchemaFuture = registerSchema(topicMetadataRequest.streamSchema.compactPrint)
+
+          val result = for {
+            r1 <- ingestFuture
+            r2 <- registerSchemaFuture
+          } yield {}
+
+          result.onComplete {
```

Comment:
Let's not use onComplete. It is meant to be a side-effect creating operation. We don't want that in actors.